cloudbuildMaster.yaml

```diff
@@ -1,7 +1,8 @@
 steps:
- - name: maven:3-jdk-8
-   entrypoint: make
-   args: ['devprep']
- - name: maven:3-jdk-8
-   entrypoint: mvn
-   args: ['package', 'appengine:deploy']
\ No newline at end of file
+ - name: 'gcr.io/cloud-builders/docker'
+   args: ['build', '-t', 'gcr.io/step-team-145/make', '.']
+ - name: 'gcr.io/step-team-145/make'
+   args: ['deploy']
+
+images:
```

Comment:
what does the `images` section mean?

Reply:
it pushes the image to the Google container registry so we can pull it whenever we want to use it

Reply:
Technically I could remove it, I don't think its completely necessary

Reply:
Oh I see. Can we rename the image to `gcr.io/step-team-145/image` or something? Naming it `make` is quite confusing.